bin/mirgecompare.py

```diff
@@ -46,7 +46,7 @@ def compare_files_vtu(first_file, second_file, file_type, tolerance = 1e-12):
         # verify individual values w/in given tolerance
         for j in range(arr1.GetSize()):
             if abs(arr1.GetValue(j) - arr2.GetValue(j)) > tolerance: 
-                print("Tolerance:", tolerance)
+                print("Tolerance:", tolerance, "Error:", abs(arr1.GetValue(j) - arr2.GetValue(j)))
                 raise ValueError("Fidelity test failed: Mismatched data array values with given tolerance")
```

Comment:
This should probably compare them all and record the max difference, then raise if the tolerance was violated.  That way we can see what the max error was.